lms/djangoapps/edxnotes/api_urls.py

```diff
@@ -6,5 +6,5 @@
 from edxnotes import views
 
 urlpatterns = [
-    url(r"^retire_user$", views.RetireUserView.as_view(), name="edxnotes_retire_user"),
+    url(r"^retire_user", views.RetireUserView.as_view(), name="edxnotes_retire_user"),
```

Comment:
I think this may be a greedier match than we want here. In other places we use `/$` to make sure it terminates at the slash. Does require the slash, though, which may be ok for this purpose. Is there a reason to keep the slash optional?

Reply:
Hm, I was copy+pasting from the forums retirement endpoint url pattern to make sure everything was consistent, but looking again I realise the vast majority of the other endpoints have `/$` like you said.